buildout.cfg

```diff
@@ -6,7 +6,8 @@ develop = .
 recipe = zc.recipe.testrunner
 eggs =
     z3c.saconfig [test]
-    pysqlite
+#    pysqlite
```

Comment:
I see you are commenting `#    pysqlite` here but then a version pin is added below. Looks strange :)